### PR TITLE
opencmp

### PR DESCRIPTION
**Pull request by opencmp**
We intent to implement opencmp as a new amp-content type.

#Intent-To-Implement
@ampproject/wg-monetization
@micajuine-ho

### CODE REVIEW

**Comment by opencmp** [comment]
@googlebot I signed it!

**Comment by opencmp** [comment]
Now it should work: https://github.com/OpenCMP-net/amphtml/commit/ae3929d387376b353e984c19a2dd88bd2c854453.patch

**Comment by micajuine-ho** [comment]
@opencmp Did you sign the CLA using this email: github@opencmp.net?

**Comment by opencmp** [comment]
> @opencmp Did you sign the CLA using this email: [github@opencmp.net](mailto:github@opencmp.net)?

I first used my business email to sign it, but changed it to the one mentioned within the contract. Is changing ok, or should I resign it?

**Comment by micajuine-ho** [comment]
@opencmp Ya lets try resigning it, and then commenting the CLA bot again.

**Comment by opencmp** [comment]
> @opencmp Ya lets try resigning it, and then commenting the CLA bot again.

I tried to resign it, but I get an error that it is already signed. So I'm not able to. Any other ideas on how to solve this one?

**Comment by micajuine-ho** [comment]
@opencmp Since this is causing so much trouble, I can just create the PR for you. Only downside is that you won't be linked as an author. Is this a good solution?

**Comment by opencmp** [comment]
> @opencmp Which emails have you used to sign it? I can check the Google CLA list to confirm that the email has be added to the white list.

for privacy reasons I just wrote you an email directly. Let me know if you received my mail.

**Comment by micajuine-ho** [comment]
> Once you've signed (or fixed any issues), please reply here with @googlebot I signed it! and we'll verify it.

@opencmp Email works. Please ping the googlebot again.

File: examples/amp-consent/cmp-vendors.amp.html [hunk 1]
```diff
@@ -102,6 +102,7 @@
           <option>Ogury</option>
           <option>SourcePoint</option>
           <option>Usercentrics</option>
+          <option>opencmp</option>
```
[the remark this answers is not in the document]
Comment (opencmp) [reply examples/amp-consent/cmp-vendors.amp.html]: done

File: examples/amp-consent/cmp-vendors.amp.html [hunk 2]
```diff
@@ -287,6 +288,21 @@ <h3>Image that is NOT blocked by consent</h3>
         </div>
       </amp-consent>
       <!-- End Usercentrics example -->
+
+      <!-- opencmp example -->
+      <amp-consent id='consent' layout='nodisplay' type='opencmp'>
```
[the remark this answers is not in the document]
Comment (opencmp) [reply examples/amp-consent/cmp-vendors.amp.html]: done

File: extensions/amp-consent/0.1/cmps.js [hunk 3]
```diff
@@ -76,3 +76,9 @@ CMP_CONFIG['Usercentrics'] = {
   'checkConsentHref': 'https://consents.usercentrics.eu/amp/checkConsent',
   'promptUISrc': 'https://amp.usercentrics.eu/amp.html',
 };
+
+CMP_CONFIG['opencmp'] = {
```
[the remark this answers is not in the document]
Comment (opencmp) [reply extensions/amp-consent/0.1/cmps.js]: done

File: extensions/amp-consent/0.1/cmps.js [hunk 4]
```diff
@@ -76,3 +82,4 @@ CMP_CONFIG['Usercentrics'] = {
   'checkConsentHref': 'https://consents.usercentrics.eu/amp/checkConsent',
   'promptUISrc': 'https://amp.usercentrics.eu/amp.html',
 };
+
```
Comment (micajuine-ho) [inline extensions/amp-consent/0.1/cmps.js:85]: Oops, looks like this our linter doesn't like this. Can you remove the new line?

Comment (opencmp) [reply extensions/amp-consent/0.1/cmps.js]: done: removed one new line

**Comment by opencmp** [comment]
@micajuine-ho Thanks for the merge. When will this be available in
```
<script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>

?

**Comment by micajuine-ho** [comment]
@opencmp  Should be in stable by June 23

**Pull request merged**